microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-studio.adoc

```diff
@@ -0,0 +1,160 @@
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `microsoft.net` in the search field.
+. Click *Microsoft .NET Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Microsoft .NET Connector:
+
+* *HTTP Listener*, which initiates a flow each time it receives a request on the configured host and port
+* *Scheduler*, which initiates a flow when a time-based condition is met
+
+For example, to configure *HTTP Listener*, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Microsoft .NET, follow these steps:
+
+. In the *Mule Palette* view, select *Microsoft DotNet* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Microsoft .NET system.
+
+To configure the global element for .Net Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, complete these fields:
++
+image::microsoft-dotnet-connection.png[Studio General tab]
++
+* *Connection*: Select a connection type.
```

Comment:
Although this one item is brief enough to justify breaking our bullet list format, the rest of the entries are not, so...:
```suggestion
* *Connection*
+
Select a connection type.
```